backend/app/models/Role.py

```diff
@@ -0,0 +1,9 @@
+from sqlalchemy import Column, Integer, String
+
+from .Base import Base
+
+
+class Role(Base):
```

Comment:
Should we add in roles right now?

I added a migration with this in my ticket since we need the roles in the db to assign them to users. I can keep it there, or if you think it belongs in this PR that would work too.

```
    op.bulk_insert(
        sa.table('roles',
            sa.Column('id', sa.Integer(), nullable=False),
            sa.Column('name', sa.String(length=80), nullable=False),
        ),
        [
            {'id': 1, 'name': 'participant'},
            {'id': 2, 'name': 'volunteer'},
            {'id': 3, 'name': 'admin'},
        ]
    )
    
```

Reply:
I think I'll move it to this PR, given that @emma-x1 can just work off this branch.

Reply:
In that case, can you please add in an auth_id field as well? It's needed for referencing the Firebase UID associated with a user in our db. I was keeping it in my ticket since that's where I initialized Firebase auth, but it's probably good to add it in here if @emma-x1 is working off this branch.

FIeld:

`auth_id = Column(String, nullable=True) # Firebase Auth ID`

Migration: 

```
def upgrade() -> None:
    # ### commands auto generated by Alembic - please adjust! ###
    op.add_column('users', sa.Column('auth_id', sa.String(), nullable=True))
    # ### end Alembic commands ###


def downgrade() -> None:
    # ### commands auto generated by Alembic - please adjust! ###
    op.drop_column('users', 'auth_id')
    # ### end Alembic commands ###
```

Reply:
Should we make `nullable=False` when Firebase auth is implemented?

Reply:
Yep, since every user should have a Firebase UID. We can just add it in as `nullable=False` right now too, should be fine.

Reply:
I think with the new edge cases you mentioned, we should keep this. From what I understand, a participant can have different types (caregiver, patient) so we can maybe assign multiple roles to that user to handle those cases. It also seems safer since stuff does seem to change around a bit whenever we talk to the npo